tests/assets/test_digital_assets.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+from ..db.conftest import inputs
+
+
+def test_asset_creation(b, user_vk):
+    asset_data = {'msg': 'hello'}
+    tx = b.create_transaction(b.me, user_vk, None, 'CREATE', asset_data=asset_data)
+    tx_signed = b.sign_transaction(tx, b.me_private)
+
+    assert b.validate_transaction(tx_signed) == tx_signed
+    assert tx_signed['transaction']['asset']['data'] == asset_data
+    assert tx_signed['transaction']['asset']['refillable'] is False
+    assert tx_signed['transaction']['asset']['divisible'] is False
+    assert tx_signed['transaction']['asset']['updatable'] is False
+    assert tx_signed['transaction']['conditions'][0]['amount'] == 1
+
+
+@pytest.mark.usefixtures('inputs')
+def test_asset_transfer(b, user_vk, user_sk):
+    tx_input = b.get_owned_ids(user_vk).pop()
+    tx_create = b.get_transaction(tx_input['txid'])
+    tx_transfer = b.create_transaction(user_vk, user_vk, tx_input, 'TRANSFER')
+    tx_transfer_signed = b.sign_transaction(tx_transfer, user_sk)
+
+    assert b.validate_transaction(tx_transfer_signed) == tx_transfer_signed
+    assert tx_transfer_signed['transaction']['asset']['id'] == tx_create['transaction']['asset']['id']
+
+
+def test_validate_bad_asset_creation(b, user_vk):
```

Comment:
Might be useful to leave a little comment at the top of each of the test blocks here to indicate what they do... didn't catch on that the validation was failing because `'divisible'` should be boolean for quite a while :blush: